lib/reversetunnel/seek/seek.go

```diff
@@ -108,10 +111,13 @@ func (s *Config) CheckAndSetDefaults() error {
 
 // Pool manages a collection of group-level seek operations.
 type Pool struct {
+	*log.Entry
```

Comment:
@russjones added a good point about not using embedded structs with many extra fields because this breaks autocomplete, let's make it `log *log.Entry` to make his life easier :)